Composer/packages/client/src/utils/storage.ts

```diff
@@ -12,15 +12,17 @@ export class ClientStorage {
     }
   }
 
-  public set<T = any>(key: string, val: T): T | void {
+  public set(key: string, val: T): T | void {
     if (val === undefined) {
       return this.remove(this.prefix(key));
     }
     this.storage.setItem(this.prefix(key), this.serialize(val));
     return val;
   }
 
-  public get<T = any>(key: string, def?: T): T {
+  public get(key: string): T | undefined;
+  public get(key: string, def: T): T;
+  public get(key: string, def?: T): T | undefined {
```

Comment:
Can we take this opportunity to rename this argument. Something like `defaultValue` would be better than `def` which is ambiguous.